templates/workspaces/airlock_manager/update_redirect_urls.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
What's different about this file compared with the one from the base workspace?
Can't we directly use it here like we do with its terraform folder?